packages/ripple-nuxt-tide/modules/publication/tide.config.js

```diff
@@ -1,53 +1,32 @@
 module.exports = {
   include: {
     publication: [
-      'field_node_documents',
       'field_node_documents.field_media_file',
-      'field_landing_page_contact',
       'field_landing_page_contact.field_paragraph_phones',
       'field_landing_page_contact.field_paragraph_social_media',
       'field_license_type',
-      'field_landing_page_component',
       'field_landing_page_component.field_paragraph_accordion',
-      'field_landing_page_component.field_complex_image_media',
       'field_landing_page_component.field_complex_image_media.field_media_image',
       'field_publication_authors'
     ],
     publicationPage: [
       'field_node_primary_site',
-      'field_node_site',
       'field_node_site.field_site_main_menu',
       'field_related_links',
-      'field_landing_page_contact',
       'field_landing_page_contact.field_paragraph_phones',
       'field_landing_page_contact.field_paragraph_social_media',
-      'field_landing_page_component',
-      'field_landing_page_component.field_paragraph_media',
       'field_landing_page_component.field_paragraph_media.field_media_image',
-      'field_landing_page_component.field_paragraph_topic',
-      'field_landing_page_component.field_timeline',
-      'field_landing_page_component.field_timeline.field_paragraph_media',
       'field_landing_page_component.field_timeline.field_paragraph_media.field_media_image',
       'field_landing_page_component.field_paragraph_accordion',
       'field_landing_page_component.field_paragraph_keydates',
-      'field_landing_page_component.field_paragraph_reference',
       'field_landing_page_component.field_paragraph_reference.field_topic',
-      'field_landing_page_component.field_paragraph_reference.field_featured_image',
       'field_landing_page_component.field_paragraph_reference.field_featured_image.field_media_image',
-      'field_landing_page_component.field_paragraph_webform',
```

Comment:
'field_landing_page_component.field_paragraph_webform' will be added by webform module.